taxi_service/urls.py

```diff
@@ -21,5 +21,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("taxi/", include("taxi.urls", namespace="taxi")),
```

Comment:
These paths have the same logic, so I think "taxi /" is unnecessary here.
To avoid error (404) just change LOGIN_REDIRECT_URL = "/"